client/src/modules/Registry/components/PersonalInfo/index.test.tsx

```diff
@@ -0,0 +1,180 @@
+import { cleanup, fireEvent, render, screen } from '@testing-library/react';
+import { Form } from 'antd';
+import { ERROR_MESSAGES, LABELS, PLACEHOLDERS } from 'modules/Registry/constants';
+import { PersonalInfo } from './index';
+
+const mockValues = {
+  firstName: 'John',
+  lastName: 'Doe',
+  location: null,
+  primaryEmail: 'test@test.com',
+  contactsEpamEmail: 'john_doe@epam.com',
+};
+
+const setup = () =>
+  render(
+    <Form initialValues={mockValues}>
+      <PersonalInfo location={null} setLocation={jest.fn()} />
+    </Form>,
+  );
+
+describe('PersonalInfo', () => {
+  afterEach(() => {
+    cleanup();
+  });
```

Comment:
```suggestion
```

Do not use `cleanup`
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-cleanup